ios/Classes/Convert.swift

```diff
@@ -120,4 +120,48 @@ class Convert {
     class func getAltitude(zoom: Double, mapView: MGLMapView) -> Double {
         return MGLAltitudeForZoomLevel(zoom, mapView.camera.pitch, mapView.camera.centerCoordinate.latitude, mapView.frame.size)
     }
+    
+    class func interpretLineOptions(options: Any?, delegate: LineOptionsSink) {
+        guard let options = options as? [String: Any] else { return }
+        
+        if let geometry = options["geometry"] as? [[Double]] {
+            delegate.setGeometry(geometry: geometry)
+        }
+        
+        if let lineJoin = options["lineJoin"] as? String {
+            delegate.setLineJoin(lineJoin: lineJoin)
+        }
+        
+        if let lineOpacity = options["lineOpacity"] as? Double {
+            delegate.setLineOpacity(lineOpacity: lineOpacity)
+        }
+        
+        if let lineColor = options["lineColor"] as? String {
+            delegate.setLineColor(lineColor: lineColor)
+        }
+        
+        if let lineWidth = options["lineWidth"] as? Double {
+            delegate.setLineWidth(lineWidth: lineWidth)
+        }
+        
+        if let lineGapWidth = options["lineGapWidth"] as? Double {
+            delegate.setLineGapWidth(lineGapWidth: lineGapWidth)
+        }
+        
+        if let lineOffset = options["lineOffset"] as? Double {
+            delegate.setLineOffset(lineOffset: lineOffset)
+        }
+        
+        if let lineBlur = options["lineBlur"] as? Double {
+            delegate.setLineBlur(lineBlur: lineBlur)
+        }
+        
+        func setLineBlur(lineBlur: Double) {
```

Comment:
remove this method

Reply:
Removed the setLineBlur.

What is the reason why you prefer the runtime api? I've taken the [Android Annotation Plugin](https://github.com/mapbox/mapbox-plugins-android/tree/master/plugin-annotation/src/main/java/com/mapbox/mapboxsdk/plugins/annotation) as a reference implementation (also contains a draggable implementation) and have been going back and forth on whether to take this approach or use the runtime api. This approach has a larger memory overhead but this way we have similar Android and iOS implementations. Curious about your thoughts on this.